src/actions/index.js

```diff
@@ -0,0 +1,9 @@
+export const createBook = (book) => ({
+  type: 'CREATE_BOOK',
```

Comment:
[Optional] It is good practice to store action type constants in a separate file.